src/SFA.DAS.ApplyService.Web/Services/OverallOutcomeService.cs

```diff
@@ -97,38 +98,48 @@ public ApplicationSummaryViewModel BuildApplicationSummaryViewModel(Apply applic
                 return model;
             }
 
-        public async Task<ApplicationSummaryWithModeratorDetailsViewModel> BuildApplicationSummaryViewModelWithModerationDetails(Apply application, string emailAddress)
+        public async Task<ApplicationSummaryWithModeratorDetailsViewModel> BuildApplicationSummaryViewModelWithGatewayAndModerationDetails(Apply application, string emailAddress)
         {
             var applicationData = application.ApplyData.ApplyDetails;
 
             var oversightReview = await _apiClient.GetOversightReview(application.ApplicationId);
 
-            var applicationUnsuccessfulModerationFail = false;
-            var applicationUnsuccessfulModerationPassOverturned = false;
-            var applicationUnsuccessfulModerationPassAndApproved = false;
+            var moderationFailedAndApproved = false;
+            var moderationPassOverturnedToFail = false;
+            var moderationPassedAndApproved = false;
+            var gatewayPassOverturnedToFail = false;
+            var moderationFailedAndOverturned = false;
             if (application?.GatewayReviewStatus == GatewayAnswerStatus.Pass)
             {
-                if (application?.ModerationStatus != null
-                    && application?.ModerationStatus == ModerationStatus.Fail
-                    && oversightReview.ModerationApproved.HasValue
-                    && oversightReview.ModerationApproved == true)
+                if (application?.ModerationStatus != null && application?.ModerationStatus == ModerationStatus.Fail)
                 {
-                    applicationUnsuccessfulModerationFail = true;
+                    if (oversightReview.ModerationApproved.HasValue && oversightReview.ModerationApproved == true)
+                    {
+                        moderationFailedAndApproved = true;
+                    }
+
+                    if (oversightReview.ModerationApproved.HasValue && oversightReview.ModerationApproved == false)
+                    {
+                        moderationFailedAndOverturned = true;
+                    }
```

Comment:
This took me a few moments to figure out what was happening. How about this to cover both cases:

`if (oversightReview.ModerationApproved.HasValue)
 {
        moderationFailedAndApproved = oversightReview.ModerationApproved;
 }`